package.json

```diff
@@ -54,9 +52,9 @@
     "jest-cli": "^18.1.0",
     "lint-staged": "^3.3.0",
     "nps": "*",
+    "nps-utils": "^1.1.0",
     "opt-cli": "^1.5.1",
     "prettier-eslint-cli": "^3.0.0",
-    "rimraf": "^2.6.0",
```

Comment:
We don't need this anymore because `nps-utils` takes care of it for us (it actually still uses `rimraf`, but it brings in the dep for us).